src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub-ssh/sftp.tf

```diff
@@ -90,7 +90,8 @@ resource "kubernetes_deployment" "jupyterhub-sftp" {
         volume {
           name = "secrets"
           secret {
-            secret_name = kubernetes_secret.jupyterhub-sftp.metadata.0.name
+            secret_name  = kubernetes_secret.jupyterhub-sftp.metadata.0.name
+            default_mode = "0600"
```

Comment:
Something changed and I'm not sure where but I needed to explicitly restrict the permissions on this file due to the following error:

```
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions 0644 for '/etc/jupyterhub-sftp/config/hostKey' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
Unable to load host key "/etc/jupyterhub-sftp/config/hostKey": bad permissions
Unable to load host key: /etc/jupyterhub-sftp/config/hostKey
sshd: no hostkeys available -- exiting.
Stream closed EOF for dev/nebari-jupyterhub-sftp-5c7b95d76f-gmdps (jupyterhub-sftp)
```

Reply:
Maybe the new version of OpenSSH refuses to run if the file has the wrong permissions now where the old one only raised a warning